src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/RefreshEmployerLevyData/RefreshEmployerLevyDataCommandHandler.cs

```diff
@@ -75,6 +82,25 @@ public class RefreshEmployerLevyDataCommandHandler : AsyncRequestHandler<Refresh
             }
         }
 
+        /// <summary> 
+        /// If there are any Submissions from Hmrc that have the same submission Id, we should discard all 
+        /// but the first. 
+        /// </summary> 
+        private DasDeclaration[] FilterDuplicateHmrcDeclarations(string empRef,
+            DasDeclaration[] declarations)
+        { 
+            var duplicateIds = declarations.GroupBy(d => d.SubmissionId).Where(g => g.Count() > 1)
+                .Select(s => s.First().SubmissionId).ToList();
+
+            var submissionIds = new StringBuilder() ;
+            duplicateIds.ForEach(submissionId =>
+                submissionIds.AppendFormat("{0}, ", submissionId));
+
+            _logger.Info($"PayeScheme '{empRef}' has duplicate submission id(s) from Hmrc = {submissionIds}");
```

Comment:
It's might be worth adding single quotes around the parameter i.e. `'{submissionIds}'`.